sklearn/tests/test_metaestimators.py

```diff
@@ -151,3 +159,108 @@ def score(self, X, y, *args, **kwargs):
+                                          LogisticRegression())
+                param_grid = {"logisticregression__C": [0.1, 1.0]}
+
+            if "param_grid" in sig or "param_distributions" in sig:
+                # SearchCV estimators
+                yield Estimator(estimator, param_grid)
+            else:
+                yield Estimator(estimator)
+
+        elif "estimators" in sig:
+            # stacking, voting
+            if issubclass(Estimator, RegressorMixin):
+                estimator = [
+                    ("est1", make_pipeline(TfidfVectorizer(),
+                                           Ridge(alpha=0.1))),
+                    ("est2", make_pipeline(TfidfVectorizer(),
+                                           Ridge(alpha=1))),
+                ]
+            else:
+                estimator = [
+                    ("est1", make_pipeline(TfidfVectorizer(),
+                                           LogisticRegression(C=0.1))),
+                    ("est2", make_pipeline(TfidfVectorizer(),
+                                           LogisticRegression(C=1))),
+                ]
+            yield Estimator(estimator)
+
+        else:
+            continue
+
+
+# TODO: remove data validation for the following estimators
+# They should be able to work on any data and delegate data validation to
+# their inner estimator(s).
+DATA_VALIDATION_META_ESTIMATORS_TO_IGNORE = [
+        "AdaBoostClassifier",
+        "AdaBoostRegressor",
+        "BaggingClassifier",
+        "BaggingRegressor",
+        "CalibratedClassifierCV",
+        "ClassifierChain",
+        "IterativeImputer",
+        "MultiOutputClassifier",
+        "MultiOutputRegressor",
+        "OneVsOneClassifier",
+        "OutputCodeClassifier",
+        "RANSACRegressor",
+        "RFE",
+        "RFECV",
+        "RegressorChain",
+        "SelfTrainingClassifier",
+        "SequentialFeatureSelector"  # not applicable (2D data mandatory)
+]
+
+DATA_VALIDATION_META_ESTIMATORS = [
+    est for est in _generate_meta_estimator_instances_with_pipeline() if
+    est.__class__.__name__ not in DATA_VALIDATION_META_ESTIMATORS_TO_IGNORE
+]
+
+
+def _get_meta_estimator_id(estimator):
+    return estimator.__class__.__name__
+
+
+@pytest.mark.parametrize(
+    "estimator", DATA_VALIDATION_META_ESTIMATORS, ids=_get_meta_estimator_id
+)
+def test_meta_estimators_delegate_data_validation(estimator):
+    # Check that meta-estimators delegate data validation to the inner
+    # estimator(s).
+    rng = np.random.RandomState(0)
+    set_random_state(estimator)
+
+    n_samples = 30
+    X = rng.choice(np.array(["aa", "bb", "cc"], dtype=object), size=n_samples)
+
+    if is_regressor(estimator):
+        y = rng.normal(size=n_samples)
+    else:
+        y = rng.randint(3, size=n_samples)
+
+    X = _enforce_estimator_tags_x(estimator, X)
+    y = _enforce_estimator_tags_y(estimator, y)
+
+    estimator.fit(X, y)
```

Comment:
```suggestion
    # Calling fit should not raise any data validation exception since X is a
    # valid input datastructure for the first step of the pipeline passed as
    # base estimator to the meta estimator.
    estimator.fit(X, y)
```